src/main/resources/i18n/messages_ru.properties

```diff
@@ -205,6 +205,7 @@ game.full.action = Нажмите, чтобы открыть игру
 game.mods.twoAndMore = {0} и {1,number,\#} подробнее
 game.gameStatus = Статус
 game.gameStatus.playing = Играет
+game.gameStatus.hosting = Хостит игру
```

Comment:
no translations are ever made on github!!! poeditor.com

Reply:
Sorry. Now I understand how it works. I will remove all changes excluding the `messages.properties` file.